SPECS/CUnit/CUnit.spec

```diff
@@ -0,0 +1,197 @@
+%global tarver 2.1-3
+
+Name:           CUnit
+Version:        2.1.3
+Release:        23%{?dist}
+Summary:        Unit testing framework for C
+Vendor:         Microsoft
+Distribution:   Mariner
+License:        LGPLv2+
+URL:            http://cunit.sourceforge.net/
+#Source0:       https://downloads.sourceforge.net/cunit/%{name}-%{tarver}.tar.bz2
+Source0:        %{name}-%{version}.tar.bz2
+
+BuildRequires:  automake
+BuildRequires:  libtool
+
+%description 
+CUnit is a lightweight system for writing, administering,
+and running unit tests in C.  It provides C programmers a basic
+testing functionality with a flexible variety of user interfaces.
+
+%package devel
+Summary:        Header files and libraries for CUnit development
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel 
+The %{name}-devel package contains the header files
+and libraries for use with CUnit package.
+
+%prep
+%setup -q -n %{name}-%{tarver}
+find -name *.c -exec chmod -x {} \;
+
+%build
+autoreconf -f -i
+%configure --disable-static
+make %{?_smp_mflags}
+
+%install
+make install DESTDIR=%{buildroot}
+rm -f `find %{buildroot} -name *.la`
+
+# work around bad docdir= in doc/Makefile*
+mkdir -p %{buildroot}%{_docdir}/%{name}
+mv %{buildroot}%{_prefix}/doc/%{name} %{buildroot}%{_docdir}/%{name}/html
+
+# add some doc files into the buildroot manually (#1001276)
+for f in AUTHORS ChangeLog COPYING NEWS README TODO VERSION ; do
+    install -p -m0644 -D $f %{buildroot}%{_docdir}/%{name}/${f}
+done
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%{_datadir}/%{name}/
+%{_libdir}/libcunit.so.*
+%dir %{_docdir}/%{name}
+%{_docdir}/%{name}/AUTHORS
+%{_docdir}/%{name}/ChangeLog
+%{_docdir}/%{name}/COPYING
```

Comment:
Please use the `%license` macro for "COPYING". Same for other license-related files (if there are any more).

Reply:
Done